board/templates/layout/basic.html

```diff
@@ -0,0 +1,26 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %}GameMuster{% endblock %}</title>
+    {% block  styles %}{% endblock %}
+    {% block scripts %}{% endblock %}
+    <link rel="stylesheet" type="text/css" href="{% static 'bootstrap/css/bootstrap.min.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'board/css/basic.css' %}">
```

Comment:
It's better to move these stylesheets up after the `title` tag so that these can be overriden by your custom stylesheet.